skimage/morphology/_flood_fill.py

```diff
@@ -7,9 +7,8 @@
 import numpy as np
 from warnings import warn
 
-from .extrema import (_resolve_neighborhood, _set_edge_values_inplace,
-                      _fast_pad)
-from ._util import _offsets_to_raveled_neighbors
+from ._util import (_resolve_neighborhood, _set_edge_values_inplace,
+                    _fast_pad, _offsets_to_raveled_neighbors)
```

Comment:
this will only be used in your other PR, right?

Reply:
No? Not sure what you are referring to.

`_resolve_neighborhood`, `_set_edge_values_inplace` and  `_fast_pad` are used in `_flood_fill.py` and `extrema.py`. While `_offsets_to_raveled_neighbors` are additionally used in `max_tree.py` and `watershed.py`.
#4165 will rely on `_resolve_neighborhood` and `_offsets_to_raveled_neighbors`.